channel.go

```diff
@@ -43,6 +43,9 @@ var (
 
 const (
 	ephemeralHostPort = "0.0.0.0:0"
+
+	// DefaultTraceSampleRate is the default sampling rate for traces.
+	DefaultTraceSampleRate = 1.0
```

Comment:
This is probability. I made it 1.0 so the behavior stays the same as previous versions.

I'll chat with the Python team tomorrow and find out their default.